lib/pipeline/addon/pipelines/project/route.js

```diff
@@ -0,0 +1,54 @@
+import { next } from '@ember/runloop';
```

Comment:
If you can mount into the regular app route instead of having your own, do that.  Otherwise import and re-export this (and cluster/route) instead of duplicating all the logic.